### PR TITLE
Format project using new prettier rules and run tests during CI

### PR DESCRIPTION
**Pull request by simon-johansson**
* Run front-end tests during CI builds
* Add some new rules to the `.prettierrc.js` (now same as app.encore.dev)
* Reformat project using the new rules

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
         key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
 
     - name: Build dashapp
-      run: cd encr.dev/cli/daemon/dash/dashapp && npm install && node node_modules/esbuild/install.js && npm run build
+      run: cd encr.dev/cli/daemon/dash/dashapp && npm install && npm test && node node_modules/esbuild/install.js && npm run build
```
Comment (eandre) [inline .github/workflows/go.yml:47]: It's probably safer to run the tests after the esbuild fix

**Review by eandre — COMMENTED** [review]
Awesome! One minor comment

File: .github/workflows/go.yml (hunk 1 above)
Comment (eandre) [inline .github/workflows/go.yml:47]: That is:
```suggestion
      run: cd encr.dev/cli/daemon/dash/dashapp && npm install && node node_modules/esbuild/install.js && npm test && npm run build
```

**Comment by simon-johansson** [comment]
Tests are now running before builds 🎉
image

**Review by eandre — APPROVED** [review]

**Pull request merged**